x/claims/keeper/invariants_test.go

```diff
@@ -0,0 +1,92 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/tharsis/ethermint/tests"
+
+	"github.com/tharsis/evmos/v2/x/claims/types"
+	inflationtypes "github.com/tharsis/evmos/v2/x/inflation/types"
+)
+
+func (suite *KeeperTestSuite) TestClaimsInvariant() {
```

Comment:
Please add these cases too (if it makes sense to):
* Claims inactive, invariants broken -> `expBroken` = `false`
* Claims active, invariants not broken -> `expBroken` = `false`

Reply:
done